acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/TransactionSignerParamsSupplier.java

```diff
@@ -72,23 +72,23 @@ private File createKeyFile() {
       throw new RuntimeException(e);
     }
 
-    return createJsonFile("ethsigner_keyfile", data);
+    return createTmpFile("ethsigner_keyfile", data);
   }
 
   private File createVaultAuthFile() {
-    return createJsonFile("vault_authfile", HashicorpVaultDocker.vaultToken.getBytes(UTF_8));
+    return createTmpFile("vault_authfile", HashicorpVaultDocker.vaultToken.getBytes(UTF_8));
   }
 
-  private File createJsonFile(final String tempNamePrefix, final byte[] data) {
+  private File createTmpFile(final String tempNamePrefix, final byte[] data) {
     final Path file;
     try {
-      file = Files.createTempFile(tempNamePrefix, ".json");
+      file = Files.createTempFile(tempNamePrefix, ".file");
       Files.write(file, data);
     } catch (final IOException e) {
       throw new RuntimeException(e);
     }
-    final File keyFile = file.toFile();
-    keyFile.deleteOnExit();
-    return keyFile;
+    final File tmpFile = file.toFile();
```

Comment:
file.toFile() seems a smell - maybe the variable should be filePath (or just path?)

Reply:
You are right! Changed to path!